doc/src/tutorial/gotchas.rst

```diff
@@ -142,6 +142,40 @@ discussed in more detail later.
 
 .. _tutorial_gotchas_equals:
 
+is_commutative == False:
+========================
+
+Passing commutative = False as argument in some operations, may lead to unexpected answers.
+For ex:
+	>>> x,y = symbols("x y")
+	>>> a,b = symbols("a b",commutative = True)
+	>>> c,d = symbols('c d',commutative = False)
+	>>> pprint((x**(a+b)).expand())
+	    a  b
+	   x ⋅x 
+	>>> pprint((x**(c+d)).expand())  
+	  c + d
+	 x
+	
+you are expanding c+d not (x**(c+d)).
```

Comment:
i added the last line because in the original issue @smichr said 
"Just so this doesn't get forgotten...
x**(a+b).expand() -- gotcha: you are expanding a+b not (x**(a+b))
powsimp(x**A*x**B) != x**(A+B) when A and B are non-commutative"

And by the rest of the code I'm basically trying to tell users that they can expect unexpected answers and complications if they use non commutative arguments.

Reply:
IMO the wording here (and from the issue) is misleading. It isn't expanding the expression simply because it can't be expanded. It's not really something that should be discussed in the gotchas in my opinion, as it's too specific (the gotchas are supposed to be things that most beginners are likely to run against). Maybe there can be a separate page about working noncommutative objects in general.